packages/ses/demos/challenge/main.js

```diff
@@ -141,7 +141,14 @@ lockdown();
       if (secretCode.slice(i, i + 1) !== guessedCode.slice(i, i + 1)) {
         return false;
       }
-      delayMS(10);
+      try {
+        delayMS(10);
+      } catch (err) {
+        if (err instanceof TypeError) {
+          // assume we cannot delay because of err
+        }
+        throw err;
```

Comment:
This will throw regardless of the condition. I presume you mean to invert the condition and move the throw into the block.

Reply:
Doh! Thanks for catching this. Done.